tests/shared.rs

```diff
@@ -141,11 +142,11 @@ fn massage_windows_line(line: &str) -> String {
 }
 
 fn strip_trailing_carriage_return(line: &str) -> Cow<'_, str> {
-    regex!(r#"\r$"#).replace(line, "")
+    regex!(r#"\r((?:\u{1b}\[\d+m)*)$"#).replace(line, "$1")
```

Comment:
I was seeing terminal color escape codes _after_ `\r`'s (not sure if that's expected?) so this handles that

Reply:
That's not where I'd have guessed they'd go, but why not 🤷🏻